src/queries/__tests__/labelText.test.tsx

```diff
@@ -93,3 +104,13 @@ test('getAllByLabelText, queryAllByLabelText, findAllByLabelText', async () => {
     getNoInstancesFoundMessage(NO_MATCHES_TEXT)
   );
 });
+
+test('getByLabelText supports accessibilityLabelledBy', async () => {
+  const { getByLabelText, queryByLabelText } = render(<FormInput />);
+
+  expect(getByLabelText(INPUT_LABEL).props.accessibilityLabelledBy).toEqual(
+    INPUT_LABELLED_BY
+  );
+  const textInput = queryByLabelText(/input/g);
+  expect(textInput?.props.accessibilityLabelledBy).toEqual(INPUT_LABELLED_BY);
+});
```

Comment:
Some improvements to the test:
1. Test both direct and nested reference
2. Verify the matched element directly against `getByTestId` result

```suggestion
test('getByLabelText supports accessibilityLabelledBy', async () => {
  const { getByLabelText, queryByLabelText } = render(<>
    <Text nativeID={INPUT_LABELLED_BY}>{INPUT_LABEL}</Text>
    <TextInput testID="textInput" accessibilityLabelledBy={INPUT_LABELLED_BY} />
  </>);

  expect(getByLabelText(INPUT_LABEL)).toEqual(getByTestId('textInput'));
  expect(getByLabelText(/input/)).toEqual(getByTestId('textInput'));
});

test('getByLabelText supports nested accessibilityLabelledBy', async () => {
  const { getByLabelText, queryByLabelText } = render(<>
    <View nativeID={INPUT_LABELLED_BY}>
      <Text >{INPUT_LABEL}</Text>
    </View>
    <TextInput testID="textInput" accessibilityLabelledBy={INPUT_LABELLED_BY} />
  </>);

  expect(getByLabelText(INPUT_LABEL)).toEqual(getByTestId('textInput'));
  expect(getByLabelText(/input/)).toEqual(getByTestId('textInput'));
});
```